packages/core/src/app/shipping/MultiShippingFormV2.tsx

```diff
@@ -63,8 +65,23 @@ const MultiShippingFormV2: FunctionComponent<MultiShippingFormV2Props> = ({
     } = config;
 
     const handleAddShippingDestination = () => {
-        setIsAddShippingDestination(true);
-    }
+        if (
+            consignments.length > 0 &&
+            !isAddShippingDestination &&
+            !isEveryConsignmentHasShippingOption
+        ) {
+            const errorConsignmentIndex = consignments.findIndex(
+                (consignment) => !consignment.selectedShippingOption,
+            );
```

Comment:
Due to the inconsistency of the consignment order, it is better to find the index each time to locate the unfinished consignment.

Reply:
Was thinking, will it not be always the last destination that is incomplete 🤔

Reply:
It can be like this.

<img width="680" alt="image" src="https://github.com/user-attachments/assets/b48dd5e8-d337-493b-b0b4-3a31077b3f8c">